[2023-11-23T14:29:42.885+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Projet1.Extract manual__2023-11-23T14:29:38.295895+00:00 [queued]>
[2023-11-23T14:29:42.895+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Projet1.Extract manual__2023-11-23T14:29:38.295895+00:00 [queued]>
[2023-11-23T14:29:42.896+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-23T14:29:42.910+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Extract> on 2023-11-23 14:29:38.295895+00:00
[2023-11-23T14:29:42.916+0000] {standard_task_runner.py:57} INFO - Started process 3846 to run task
[2023-11-23T14:29:42.919+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Projet1', 'Extract', 'manual__2023-11-23T14:29:38.295895+00:00', '--job-id', '578', '--raw', '--subdir', 'DAGS_FOLDER/Projet.py', '--cfg-path', '/tmp/tmpmjmwuvuf']
[2023-11-23T14:29:42.921+0000] {standard_task_runner.py:85} INFO - Job 578: Subtask Extract
[2023-11-23T14:29:42.970+0000] {task_command.py:416} INFO - Running <TaskInstance: Projet1.Extract manual__2023-11-23T14:29:38.295895+00:00 [running]> on host b36ebcee5948
[2023-11-23T14:29:43.054+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Projet1' AIRFLOW_CTX_TASK_ID='Extract' AIRFLOW_CTX_EXECUTION_DATE='2023-11-23T14:29:38.295895+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-23T14:29:38.295895+00:00'
[2023-11-23T14:29:43.285+0000] {warnings.py:109} WARNING - /opt/***/dags/Projet.py:27: DtypeWarning: Columns (0) have mixed types. Specify dtype option on import or set low_memory=False.
  df_urgences = pd.read_csv(csv_file_path_urgences, delimiter=';')

[2023-11-23T14:29:43.663+0000] {logging_mixin.py:151} INFO - 102
[2023-11-23T14:29:43.665+0000] {logging_mixin.py:151} INFO - 100
[2023-11-23T14:29:43.674+0000] {logging_mixin.py:151} INFO -    Code tranches d'age             Age
0                    0       tous Ã¢ges
1                    1         0-4 ans
2                    2        5-14 ans
3                    3       15-44 ans
4                    4       45-64 ans
5                    5       65-74 ans
6                    6  75 ans ou plus
[2023-11-23T14:29:43.675+0000] {logging_mixin.py:151} INFO - Index(['code_age', 'agess'], dtype='object')
[2023-11-23T14:29:43.676+0000] {logging_mixin.py:151} INFO - zmmmmmmarr 
[2023-11-23T14:29:43.677+0000] {logging_mixin.py:151} INFO - Index(['dep', 'date_de_passage', 'sursaud_cl_age_corona', 'nbre_pass_corona',
       'nbre_pass_tot', 'nbre_hospit_corona', 'nbre_pass_corona_h',
       'nbre_pass_corona_f', 'nbre_pass_tot_h', 'nbre_pass_tot_f',
       'nbre_hospit_corona_h', 'nbre_hospit_corona_f', 'nbre_acte_corona',
       'nbre_acte_tot', 'nbre_acte_corona_h', 'nbre_acte_corona_f',
       'nbre_acte_tot_h', 'nbre_acte_tot_f'],
      dtype='object')
[2023-11-23T14:29:43.686+0000] {base.py:73} INFO - Using connection ID 'postgres_connexion' for task execution.
[2023-11-23T14:29:43.720+0000] {base.py:73} INFO - Using connection ID 'postgres_connexion' for task execution.
[2023-11-23T14:29:43.748+0000] {base.py:73} INFO - Using connection ID 'postgres_connexion' for task execution.
[2023-11-23T14:30:04.867+0000] {base.py:73} INFO - Using connection ID 'postgres_connexion' for task execution.
[2023-11-23T14:30:04.876+0000] {sql.py:418} INFO - Running statement: select * from urgences ; , parameters: None
[2023-11-23T14:30:04.881+0000] {logging_mixin.py:151} INFO - Une erreur s'est produite : relation "urgences" does not exist
LINE 1: select * from urgences ; 
                      ^
[2023-11-23T14:30:04.882+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-23T14:30:04.892+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Projet1, task_id=Extract, execution_date=20231123T142938, start_date=20231123T142942, end_date=20231123T143004
[2023-11-23T14:30:04.956+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-23T14:30:04.976+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
